package.json

```diff
@@ -57,7 +57,13 @@
     "grunt-prompt": "~1.3.0",
     "grunt-replace": "0.6.2",
     "grunt-sass": "~1.1.0-beta",
+    "gulp": "3.8.10",
+    "gulp-iconfont": "1.0.0",
+    "gulp-iconfont-css": "0.0.9",
+    "gulp-install": "0.2.0",
+    "gulp-sketch": "0.0.7",
     "jquery": "2.1.4",
+    "jsdoc": "^3.4.0",
```

Comment:
I think the tool does not mind, the important is we have a minimum documented API.

JSDoc is really powerful and allowed to create templates with filter, ie, http://openlayers.org/en/v3.10.1/apidoc/, but if by default the output of documentation is better I agree to use it and does not spent time on the output.